docs/tasks/includes/_rust-read.md

```diff
@@ -0,0 +1,28 @@
+
+Use the [`Reader`](https://docs.rs/c2pa/latest/c2pa/struct.Reader.html) struct to read manifest data from a file or stream.
```

Comment:
This is a tangent. Please feel free to ignore.

I don't know how much control you have over the style, but these links are rendering sort of hard to read for me. The underline is too close to the text, and links are usually a different color, like blue.

![Screenshot 2025-02-14 at 12 16 48 PM](https://github.com/user-attachments/assets/1d9da88f-b9c1-4a13-ba8f-851dc154d73f)

Compare how `rustdoc` renders links.

![Screenshot 2025-02-14 at 12 18 46 PM](https://github.com/user-attachments/assets/ec47bff7-16db-4c3d-967a-1542e0a26500)

Reply:
This is an orthogonal discussion, but we can certainly look at the visual design of the site...   We do have complete control over it.

FWIW, its been this way since its launch in 2022 based on work that Kozma and Pia did.  I suspect they didn't really consider the display of links on monospace (i.e. code) font text, and I agree it could use a bit more space between the underline and text.